src/Diagram.php

```diff
@@ -132,6 +132,9 @@ public function create($entity_types) {
                 $field_property_info['type'] = 'list<' . $field_property_info['type'] . '>';
               }
 
+              $reference_target_type = NULL;
+              $target_bundles = [];
+              // Get target info from entity reference field type.
               if ($instance_info->getType() == 'entity_reference' || $instance_info->getType() == 'entity_reference_revisions') {
```

Comment:
Here is if for `$instance_info->getType()`

Reply:
updated